operator/v1/types_network.go

```diff
@@ -137,6 +146,14 @@ type NetworkMigration struct {
 	// supported features.
 	// +optional
 	Features *FeaturesMigration `json:"features,omitempty"`
+
+	// mode indicates the mode of network migration.
+	// The supported values are Live, Offline.
+	// Live: the migration operation will not cause service interruption
+	// Offline: the migration operation will cause service interruption
+	// if unset, the default mode is Offline
```

Comment:
nits:

The possible values are usually quoted.

If you put it that way, why would one ever do an Offline migration? If there is no reason other than the fact that live is new an untested, maybe we could add a bit more detail in other differences about the two methods.

Reply:
Yes, live is new and it will be only available for SD clusters. So we don't want to deprecate the offline one now. But in the long term, I think we will only support live migration.